### PR TITLE
[TEC-3623] removed duplicate register of `events-admin.css`

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[TEC-3623]
Related Ticket: https://moderntribe.atlassian.net/browse/CE-13

CSS loaded twice for the-events-calendar:
ce-13 tec duplicate

After the fix:
ce-13 tec duplicate-after

[TEC-3623]: https://moderntribe.atlassian.net/browse/TEC-3623

### CODE REVIEW

File: src/Tribe/Assets.php [hunk 1]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
-			'tribe_events-admin',
-			'events-admin.css',
-			array(),
-			'admin_enqueue_scripts',
-			array(
-				'groups'       => array( 'events-admin' ),
-				'conditionals' => array( $this, 'should_enqueue_admin' ),
-			)
-		);
+//		tribe_asset(
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:86]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: 2e21b38136658d526549c879e25b0de6</sub>

File: src/Tribe/Assets.php [hunk 2]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//			$plugin,
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:87]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: 19071771eb4af89d70caa36ce7c11bc8</sub>

File: src/Tribe/Assets.php [hunk 3]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//			'tribe_events-admin',
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:88]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: 40bd086475dfb1564f2c2949e7e3d337</sub>

File: src/Tribe/Assets.php [hunk 4]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//			'events-admin.css',
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:89]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: 05d0e5dea8758bebf0cd5f5e8bc228a6</sub>

File: src/Tribe/Assets.php [hunk 5]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//			array(),
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:90]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: b66d28f5f98d2d8bee3995a9bc4a0ba8</sub>

File: src/Tribe/Assets.php [hunk 6]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//			'admin_enqueue_scripts',
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:91]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: a8debdaab57f5335f23f00cc27e15e96</sub>

File: src/Tribe/Assets.php [hunk 7]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//			array(
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:92]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: b17115184adc5223aa8715f1e7276ca9</sub>

File: src/Tribe/Assets.php [hunk 8]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//				'groups'       => array( 'events-admin' ),
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:93]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 14; use block comment if you need indentation

<sub>hash: efb3fab5a9d93bffddadbfc3579184e7</sub>

File: src/Tribe/Assets.php [hunk 9]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//				'conditionals' => array( $this, 'should_enqueue_admin' ),
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:94]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 14; use block comment if you need indentation

<sub>hash: de7ae25d7790a7bf0071d113620a4413</sub>

File: src/Tribe/Assets.php [hunk 10]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//			)
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:95]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: 03d40dc303af6bbc27b2f4ead4563f48</sub>

File: src/Tribe/Assets.php [hunk 11]
```diff
@@ -83,17 +83,17 @@ public function register() {
...
+//		);
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:96]: * :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: f2b62fd0ff688cf52c9246848eb618be</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 23 error(s)

**Review by lucatume — COMMENTED** [review]
The base for the PR should not be `master`, but the `release/B20.09` branch.
Provide and before and after screencast of the fix for reviewers to understand what problem you're solving and make sure all still works as intended.

**Review by bordoni — APPROVED** [review]

**Comment by bordoni** [comment]
Lets get the Changelog @rafsuntaskin, it will go on B20.10.

**Pull request merged**